src/Tizen.TV.UIControls.Forms/Renderer/MediaPlayerImpl.cs

```diff
@@ -20,41 +20,61 @@
 using System.Runtime.CompilerServices;
 using System.Threading.Tasks;
 using Tizen.Multimedia;
+using Tizen.TV.Multimedia;
 using Tizen.TV.UIControls.Forms.Renderer;
 using Xamarin.Forms;
 using Xamarin.Forms.Internals;
 using Xamarin.Forms.Platform.Tizen;
-
+using TVM = Tizen.TV.Multimedia;
```

Comment:
`using TVPlayer = Tizen.TV.Multimedia.Player` and just use `TVPlayer` instead of `TVM.Player`.

Reply:
done